### ISSUE TITLE
Add-path suppport for VPN address families

### ISSUE THREAD

**Issue by adrian62** [Exa-Networks/exabgp#632]
Presently the addpath capability described in RFC7911 is not supported for VPN unicast AFs 'ipv4 mpls-vpn' and 'ipv6 mpls-vpn'. This is an enhancement request for the two AFs aforementioned.

##### ISSUE TYPE
Enhancement of functionality available for other AFs.

##### OS
Ubuntu 16.04 LTS and others

##### VERSION
```
ExaBGP : 3.4.18
Python : 2.7.11+ (default, Apr 17 2016, 14:00:29)  [GCC 5.3.1 20160413]
Uname  : #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
```

##### ENVIRONMENT 
```
[exabgp.daemon]
user = 'exabgp'
```

##### CONFIGURATION
```
group EXA_AFX2_2_IAVRR {
        local-as 65060;
        peer-as 65060;
        router-id 10.0.99.9;
        local-address 10.0.99.9;
#
        process inject-routes {
                run /home/AFX/qafx.py;
       }
        neighbor 10.0.99.4{
                description "EXA_AFX2_2_IAVRR";
                hold-time 90;
                add-path send;
                family {
                        ipv4 mpls-vpn;
                        ipv6 mpls-vpn;
                }
        }

```

##### IMPORTANCE
blocker for utilizing this application in a production environment

**Comment by thomas-mangin** [comment]
provided a patch offline - waiting for feedback

**Comment by thomas-mangin** [comment]
mpls-vpn allowed for add-path on master with
https://github.com/Exa-Networks/exabgp/commit/f7043a8306056f279f2f46a73d9b552ea75303c9

Waiting for feedback for 3.4

however ExaBGP will send non-add path routes on an add-path session if 'path-information' is omitted. I would assume a default path-information should always be sent / the RFC need checking.

**Comment by thomas-mangin** [comment]
and backported to the now released 3.4.19.
